src/rules/comment-empty-line-before/__tests__/index.js

```diff
@@ -107,3 +108,18 @@ testRuleScss("always", tr => {
 testRuleScss("never", tr => {
   tr.ok("a { color: pink;\n\n// comment\ntop: 0; }", "single-line comment ignored")
 })
+
+const testRuleLess = ruleTester(rule, ruleName, {
+  postcssOptions: {
+    syntax: lessSyntax,
+  },
+})
+
+testRuleLess("always", tr => {
+  tr.ok("a { color: pink;\n// comment\ntop: 0; }", "single-line comment ignored")
+  tr.ok("// first line\n// second line\na { color: pink; }", "subsequent single-line comments ingnored")
+})
+
+testRuleLess("never", tr => {
+  tr.ok("a { color: pink;\n\n// comment\ntop: 0; }", "single-line comment ignored")
+})
```

Comment:
You might want to installed an `editorconfig` [plugin](http://editorconfig.org/#download) so end of file newlines are automatically added.